### PR TITLE
Fix cpuload

### PR DESCRIPTION
**Pull request by haslinghuis**
`cpuload` was not updated as the `update_live_status` was not running as the `onConnect` function didn't allow running asynchronous. This fix also brings back the quad status and battery legend.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

File: src/js/serial_backend.js
Comment (chmelevskij) [inline src/js/serial_backend.js:694]: There is no awai, no reason to mark it as async
```suggestion
function getStatus() {
```

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
